### PR TITLE
docs: improve placeholder text of the route header override

### PR DESCRIPTION
**Pull request by okaybase**
Please answer these questions before submitting a pull request, **or your PR will get closed**.

**Why submit this pull request?**

1. Improve placeholder text of the route header override.
2. Improve label of the route match parameter value.
3. Correct the title for creating route in the file search-route.spec.js.
4. Correct the spelling of some words.

- [ ] Bugfix
- [ ] New feature provided
- [ ] Improve performance
- [x] Improve document
- [ ] Backport patches

**What changes will this PR take into?**

Please update this section with detailed description.

**Related issues**

fix/resolve

image

image

**Checklist:**

- [x] Did you explain what problem does this PR solve? Or what new features have been added?
- [ ] Have you added corresponding test cases?
- [x] Have you modified the corresponding document?
- [x] Is this PR backward compatible? If it is not backward compatible, please discuss on the mailing list first

### CODE REVIEW

File: web/src/pages/Route/locales/en-US.ts [hunk 1]
```diff
@@ -48,7 +48,7 @@ export default {
 
   'page.route.form.itemRulesRequiredMessage.parameterName':
     'Only letters and Numbers are supported, and can only begin with letters',
-  'page.route.value': 'Value',
+  'page.route.value': 'Parameter Value',
```
Comment (iamayushdas) [inline web/src/pages/Route/locales/en-US.ts:51]: Why to change this, `Value` is fine IMO

Comment (okaybase) [reply web/src/pages/Route/locales/en-US.ts:51]: Yes, without the Parameter is also ok here. @iamayushdas 
Parameter is added only to be consistent with the Chinese version.

image

**Review by iamayushdas — APPROVED** [review]

**Comment by iamayushdas** [comment]
cc @juzhiyuan have a look here

**Comment by okaybase** [comment]
Thanks very much for your careful review! @iamayushdas

**Review by gxthrj — APPROVED** [review]

**Comment by okaybase** [comment]
Please take time to review, thanks! @juzhiyuan @nic-chen

**Review by nic-chen — APPROVED** [review]

**Review by juzhiyuan — APPROVED** [review]

**Pull request merged**